File: yukicoder/9xxx/tyama_yukicoder9001.cs
using System;
class Test{
	const int SIZE=9999999;
	static byte[] z=new byte[SIZE];
	static int input_count=0;

	static int get(){
		int r=0;
		for(;'0'<=z[input_count]&&z[input_count]<='9';)r=r*10+z[input_count++]-'0';
		input_count++;
		return r;
	}
	static int getC(){
		return z[input_count++];
	}
	static void Main(){
		Console.OpenStandardInput().Read(z,0,SIZE);
		Console.Write(get()+get());
		Console.Write((char)32);
		for(int c;(c=getC())>0;)Console.Write((char)c);
	}
}

File: yukicoder/9xxx/tyama_yukicoder9001.c
#include <stdio.h>
char z[999999];
static int input_count=0;

int get(){
	int r=0;
	for(;'0'<=z[input_count]&&z[input_count]<='9';)r=r*10+z[input_count++]-'0';
	input_count++;
	return r;
}
int getC(){
	return z[input_count++];
}
int main(){
	fread(z,1,sizeof(z),stdin);
	printf("%d ",get()+get());
    int c;
	for(;(c=getC())>0;)putchar(c);
}